src/js/ScrollableMessageButtons.js

```diff
@@ -1,13 +1,10 @@
 import React from 'react';
 import SoftButtonImage from './Templates/Shared/SoftButtonImage';
 import uiController from './Controllers/UIController';
-import { updateResetPeriod, resetTimeout } from './actions';
 import store from './store.js';
-import { connect } from 'react-redux'
-import { DEFAULT_RESET_TIMEOUT } from "./Alert"
 
 const OUT_OF_BOUND_RESET_PERIOD = 1000000;
-class ScrollableMessageButtons extends React.Component {
+export default class ScrollableMessageButtons extends React.Component {
```

Comment:
@LiliiaShlikhtLuxoft Why `export deafult` ?

Reply:
@ValeriiMalkov because every class should be exported, but previous commit changed it and put it at the end of class (for props purpose). I returned it.